lib/assert.js

```diff
@@ -206,13 +206,11 @@ function getErrMessage(call) {
   }
 }
 
-function innerOk(args, fn) {
-  var [value, message] = args;
-
+function innerOk(fn, argc, value, message) {
```

Comment:
Nit: argc is not really commonly used in JS and not everyone might know the terminology. So I suggest to switch to argLength.

Reply:
updated parameter name